Update section.md (#1235)
Codecademy/docs: content/html/concepts/semantic-html/terms/section/section.md

Tags:
  - 'Semantic'
  - 'Tags'
  - 'Elements'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

> The **`<section>`** semantic HTML element defines elements in a document, such as chapters, headings, or any other area of the document with the same theme.

Reviewer: ```suggestion
The **`<section>`** HTML element defines elements in a document, such as chapters, headings, or any other area of the document with the same theme.
```

For example, content with the same theme such as articles about cricket can go under a single `<section>`. A website’s home page could be split into sections for the introduction, news items, and contact information.

## Syntax

```pseudo
<section>

>   <!-- Elements in a document with the same theme -->

Reviewer: ```suggestion
  <!-- Elements in a document with a common theme -->
```

Tags:
  - 'Semantic'
  - 'Tags'
  - 'Elements'
CatalogContent:
  - 'learn-html'
  - 'paths/front-end-engineer-career-path'
---

> The **`<section>`** element defines elements in a document, such as chapters, headings or any other area of the document with a common theme.

Reviewer: ```suggestion
The **`<section>`** element defines elements in a document, such as chapters, headings, or any other area of the document with a common theme.
```

For example, content with the same theme such as articles about cricket can go under a single `<section>`. A website’s home page could be split into sections for the introduction, news items, and contact information.

## Syntax

```pseudo
<section>

>   <!-- Elements in a document with a common theme -->

Reviewer: ```suggestion
  Elements in a document with a common theme
```

  </article>
</section>
```
  

> There are also examples where we can use other elements that semantically fit better then `<section>` element.

Reviewer 2: These examples belong better in their own example section after the existing example section, rather than in the syntax section. Perhaps with the header "Alternatives to `<section>`"

> - The first example is the `<header>` element which can be used to describe that the content inside is the header of the page.

Reviewer 2: Because of the interspersed code blocks, this doesn't really work as a bulleted list. I'd remove the bullets and have the code blocks introduced with regular paragraphs.

  
```html
<!DOCTYPE html>
<html>
  <head></head>
  <body>
    <!-- Header of the page -->
    <header>
      <h1>Welcome to my site!</h1>
    </header>
      
    <!-- Content of the page -->
    <section></section>
    <section></section>
  </body>
</html>
```
- Second example is the `<nav>` element. It is used to describe that the content inside is a navigation system of the page.  
  
```html
<!DOCTYPE html>
<html>
  <head></head>
  <body>
    <!-- Navigation  of the page -->
    <nav>
      <ul>
        <li>Home</li>
        <li>Store</li>
        <li>Profile</li>
      </ul>
    </nav>
    
    <!-- Content of the page -->
    <section></section>
    <section></section>
  </body>
</html>
```
  
- The third example is the `<main>` element which is used to describe that the content inside is the main content of the page.

> ```html

Reviewer 2: Make sure there's a line between paragraphs and the code blocks.

  </body>
</html>
```

> ## Alternatives to `<section>`

Reviewer: @D-Salkovic upon second look, we should turn these into bullets (without the html code block that follows) and link out to the respective entries on each element like:

There are also examples where we can use other elements that semantically fit better than the `<section>` element:
  
- The [`<header>`](https://www.codecademy.com/resources/docs/html/semantic-html/header) element which can be used to describe that the content inside is the header of the page.
- The [`<nav>`](https://www.codecademy.com/resources/docs/html/semantic-html/nav) element. It is used to describe that the content inside is a navigation system of the page.
- The [`<main>`](https://www.codecademy.com/resources/docs/html/semantic-html/main) element which is used to describe that the content inside is the main content of the page.
- The [`<footer>`](https://www.codecademy.com/resources/docs/html/semantic-html/footer) element which is used to describe that the content inside is the footer of the page.

> ## Alternatives to `<section>`

Reviewer 3: ```suggestion
## Alternative Elements
```

> There are also examples where we can use other elements that semantically fit better then `<section>` element.

Reviewer: ```suggestion
There are also examples where we can use other elements that semantically fit better than `<section>` element.
```

> There are also examples where we can use other elements that semantically fit better then the `<section>` element.

Reviewer: ```suggestion
There are also examples where we can use other elements that semantically fit better than the `<section>` element.
```

> There are also examples where we can use other elements that semantically fit better than the `<section>` element.

Reviewer 3: ```suggestion
The `<section>` element should only be used if no other suitable element is available. Some alternatives include the following:
```

> - The [`<header>`](https://www.codecademy.com/resources/docs/html/semantic-html/header) element which can be used to describe that the content inside is the header of the page.

Reviewer 3: ```suggestion
- The [`<header>`](https://www.codecademy.com/resources/docs/html/semantic-html/header) element describes that the content inside is the header of the page.
```

> 
> ```html
> <!DOCTYPE html>
> <html>
>   <head></head>
>   <body>
>     <!-- Header of the page -->
>     <header>
>       <h1>Welcome to my site!</h1>
>     </header>
> 
>     <!-- Content of the page -->
>     <section></section>
>     <section></section>
>   </body>
> </html>
> ```
> 

Reviewer 3: ```suggestion
```

> Second example is the `<nav>` element. It is used to describe that the content inside is a navigation system of the page.  

Reviewer: ```suggestion
The second example is the `<nav>` element. It is used to describe that the content inside is a navigation system of the page.
```

> - The [`<nav>`](https://www.codecademy.com/resources/docs/html/semantic-html/nav) element. It is used to describe that the content inside is a navigation system of the page.

Reviewer 3: ```suggestion
- The [`<nav>`](https://www.codecademy.com/resources/docs/html/semantic-html/nav) element describes that the content inside is a navigation system of the page.
```

  
```html
<!DOCTYPE html>
<html>
  <head></head>
  <body>

>     <!-- Navigation  of the page -->

Reviewer: ```suggestion
    <!-- Navigation of the page -->
```

> 
> ```html
> <!DOCTYPE html>
> <html>
>   <head></head>
>   <body>
>     <!-- Navigation of the page -->
>     <nav>
>       <ul>
>         <li>Home</li>
>         <li>Store</li>
>         <li>Profile</li>
>       </ul>
>     </nav>
> 
>     <!-- Content of the page -->
>     <section></section>
>     <section></section>
>   </body>
> </html>
> ```
> 

Reviewer 3: ```suggestion
```

> - The [`<main>`](https://www.codecademy.com/resources/docs/html/semantic-html/main) element which is used to describe that the content inside is the main content of the page.

Reviewer 3: ```suggestion
- The [`<main>`](https://www.codecademy.com/resources/docs/html/semantic-html/main) describes that the content inside is the main content of the page.
```

> 
> ```html
> <!DOCTYPE html>
> <html>
>   <head></head>
>   <body>
>     <!-- Header of the page -->
>     <header>
>       <h1>Welcome to my site!</h1>
>     </header>
> 
>     <!-- Main content of the page -->
>     <main>
>       <section></section>
>       <section></section>
>     </main>
>   </body>
> </html>
> ```
> 

Reviewer 3: ```suggestion
```

> - The [`<footer>`](https://www.codecademy.com/resources/docs/html/semantic-html/footer) element which is used to describe that the content inside is the footer of the page.

Reviewer 3: ```suggestion
- The [`<footer>`](https://www.codecademy.com/resources/docs/html/semantic-html/footer) element describes that the content inside is the footer of the page.
```

> 
> ```html
> <!DOCTYPE html>
> <html>
>   <head></head>
>   <body>
>     <!-- Header of the page -->
>     <header>
>       <h1>Welcome to my page!</h1>
>     </header>
> 
>     <!-- Main content of the page -->
>     <main>
>       <section></section>
>       <section></section>
>     </main>
> 
>     <!-- Footer of the page -->
>     <footer>
>       <p>by: User</p>
>       <span>email: user@email.com</span>
>     </footer>
>   </body>
> </html>
> ```

Reviewer 3: ```suggestion
```
